SimpleInventoryManagementSystem/Program.cs

```diff
@@ -1,13 +1,63 @@
 ﻿
+using SimpleInventoryManagementSystem;
 using System;
 
-namespace SimpleInventoryManagementSystem
+namespace SimpleInventoryManagementSystem;
+
+public class Program
 {
-    public class Program
+    enum Services{
+        Exit,
+        AddProduct,
+        ViewProducts
+    }
+
+    public static void PrintServices()
     {
-        public static void Main(string[] args)
+        Console.WriteLine("Choose The number of service you want from our menu bellow");
+        foreach (Services service in Enum.GetValues(typeof(Services)))
         {
-            
+            Console.WriteLine($"{service} is {(int)service}");
+        }
+    }
+    public static void Main(string[] args)
+    {
+        Inventory inventory = new Inventory(new List<Product>());
+
+        while (true)
+        {
+            PrintServices();
+
+            var input = Console.ReadLine();
+            if (!int.TryParse(input,out int number))
+            {
+                throw new ArgumentException("invalid input");
+            }
```

Comment:
Let's just trigger another loop if the input is incorrect. This is recoverable and the program can work if the user tries again.

You are also losing strong typing by not using your enumeration, we can make the code more readable with the enum as follows:
 
```suggestion
            if (!Enum.TryParse<Services>(input, out var option))
            {
                Console.WriteLine("Invalid input. Please try again");
                continue;
            }
```